project_euler/problem_006/sol1.rb

```diff
@@ -0,0 +1,35 @@
+#Project Euler Problem 6: #https://projecteuler.net/problem=6
+
+#Sum square difference
+
+#The sum of the squares of the first ten natural numbers #is,
+#    1^2 + 2^2 + ... + 10^2 = 385
+#The square of the sum of the first ten natural numbers #is,
+#    (1 + 2 + ... + 10)^2 = 55^2 = 3025
+#Hence the difference between the sum of the squares of #the first ten
+#natural numbers and the square of the sum is 3025 - 385 = 2640.
+#Find the difference between the sum of the squares of the first one
+#hundred natural numbers and the square of the sum.
+
+def solution?(num)
```

Comment:
It looks like project euler scripts follow specific guidelines.
Could you please double-check those? https://github.com/TheAlgorithms/Ruby/blob/master/project_euler/README.md

Thank you